src/Microsoft.AspNetCore.Sockets.Client/Format.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Sockets.Client
+{
+    public enum Format
```

Comment:
Hmm. We have a SignalR.Common we use for sharing types like this. Maybe we need a Sockets.Common?